### PR TITLE
Added appBarTheme in startup_namer project

### PR DESCRIPTION
**Pull request by itsjatinnagar**
Project _startup_namer_ has issue with **AppBar** - its _backgroundColor_ is not changing to white by using _primaryColor_ property.
I updated the code to use **appBarTheme** to give desired appearance to the app bar.

### CODE REVIEW

**Comment by itsjatinnagar** [comment]
> Thanks for your pull request. It looks like this may be your first contribution to a Google open source project (if not, look below for help). Before we can look at your pull request, you'll need to sign a Contributor License Agreement (CLA).
> 
> 📝 **Please visit https://cla.developers.google.com/ to sign.**
> 
> Once you've signed (or fixed any issues), please reply here with `@googlebot I signed it!` and we'll verify it.
> 
> #### What to do if you already signed the CLA
> ##### Individual signers
> * It's possible we don't have your GitHub username or you're using a different email address on your commit. Check [your existing CLA data](https://cla.developers.google.com/clas) and verify that your [email is set on your git commits](https://help.github.com/articles/setting-your-email-in-git/).
> 
> ##### Corporate signers
> * Your company has a Point of Contact who decides which employees are authorized to participate. Ask your POC to be added to the group of authorized contributors. If you don't know who your Point of Contact is, direct the Google project maintainer to [go/cla#troubleshoot](http://go/cla#troubleshoot) ([Public version](https://opensource.google/docs/cla/#troubleshoot)).
> * The email used to register you as an authorized contributor must be the email used for the Git commit. Check [your existing CLA data](https://cla.developers.google.com/clas) and verify that your [email is set on your git commits](https://help.github.com/articles/setting-your-email-in-git/).
> * The email used to register you as an authorized contributor must also be [attached to your GitHub account](https://github.com/settings/emails).
> 
> ℹ️ **Googlers: [Go here](https://goto.google.com/prinfo/https%3A%2F%2Fgithub.com%2Fflutter%2Fcodelabs%2Fpull%2F186) for more info**.

@googlebot I signed it!

**Review by domesticmouse — APPROVED** [review]

File: startup_namer/step8_themes/lib/main.dart [hunk 1]
```diff
@@ -15,7 +15,11 @@ class MyApp extends StatelessWidget {
     return MaterialApp(
       title: 'Startup Name Generator',
       theme: ThemeData(
-        primaryColor: Colors.white,
+        // primaryColor: Colors.white,  /// Does not work for AppBar
```
Comment (domesticmouse) [inline startup_namer/step8_themes/lib/main.dart:18]: Any chance of deleting this line? Thanks!

Comment (itsjatinnagar) [reply startup_namer/step8_themes/lib/main.dart]: Sure @domesticmouse 
I have deleted the commented line and pushed a new commit.
Please review it.
Thanks...

Comment (domesticmouse) [reply startup_namer/step8_themes/lib/main.dart]: Thank you for updating your PR. I will land it as soon as the CI turns green.

Comment (domesticmouse) [reply startup_namer/step8_themes/lib/main.dart]: Actually, it looks like this repo now requires signed commits. If you still want to submit this PR, please feel free to re-manufacture it using https://docs.github.com/en/codespaces/the-githubdev-web-based-editor instead

Comment (itsjatinnagar) [reply startup_namer/step8_themes/lib/main.dart]: @domesticmouse Please review I have pushed the signed commits to the branch.

Comment (domesticmouse) [reply startup_namer/step8_themes/lib/main.dart]: Github still appears to be upset with this PR. Le sigh

Comment (itsjatinnagar) [reply startup_namer/step8_themes/lib/main.dart]: Is it ok, If I close this pull request and create a new one

Comment (domesticmouse) [reply startup_namer/step8_themes/lib/main.dart]: Absolutely, apologies this is so difficult

Comment (itsjatinnagar) [reply startup_namer/step8_themes/lib/main.dart]: No worries

Comment (itsjatinnagar) [reply startup_namer/step8_themes/lib/main.dart]: @domesticmouse Please review my new PR #189 
Thanks

**Pull request closed without merging**